tests/foreman/ui/test_ldap_auth.py

```diff
@@ -119,13 +133,34 @@ def test_positive_ad_basic_no_roles(self):
         :expectedresults: Log in to foreman UI successfully but cannot access
             functional areas of UI
 
-        :caseautomation: notautomated
-
         :CaseLevel: System
         """
+        server_name = gen_string('alpha')
+        with Session(self) as session:
+            make_ldapauth(
```

Comment:
I think that creation of auth source can definitely be moved to setupclass section.
Also, I think that cases in general can be moved to:
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_user.py#L1287
or case from test_user file can be moved here